addons/main/functions/fnc_executeForPlayersInRange.sqf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Gets all players within a radius around the given object and executes a given function, command or script with arguments on these clients.
+ *
+ * Arguments:
+ * 0: Range <NUMBER>
+ * 1: Object <OBJECT>
+ * 2: Function <STRING> or <CODE>
+ * 3: Arguments <ANY>
+ * 
+ * Return:
+ * Nothing
+ */
+
+ params ["_range", "_object", "_fnc", "_args"];
+
+private _playersInRange = [_range, _object] call AE3_main_fnc_getPlayersInRange;
+
+// using remoteExec on a zero sized array leads to RPT spam warnings
+if ((count _playersInRange) == 0) exitWith {};
+
+// use remoteExec if _fnc is type "string"
```

Comment:
I don't think this is necessary. If a function should be executed scheduled, then this can be used like
```sqf
[3, _computer, {_this spawn AE3_armaos_fnc_terminal_uiOnTex_updateOutput}, [_computer, _output]] call AE3_main_fnc_executeForPlayersInRange;
```

Reply:
What do you exactly mean with "this" when you say "I don't think 'this' is necessary."